.gitignore

```diff
@@ -23,7 +23,7 @@ var/
 # Usually these files are written by a python script from a template
 # before PyInstaller builds the exe, so as to inject date/other infos into it.
 *.manifest
-*.spec
+#*.spec
```

Comment:
Why?

Reply:
We have an [installer](https://github.com/SiLab-Bonn/testbeam_analysis/blob/gui/scripts/pyinstaller_linux.spec) for TBA. It works under Linux after some heavy hacking... Windows fails.
Executable here: https://github.com/SiLab-Bonn/testbeam_analysis/releases/download/0.0.1/Linux_Binary.bz2.run

Reply:
Although this needs clean up. I see my path hardcoded there ;-)